### PR TITLE
Summer 2019

### PR DESCRIPTION
**Pull request by agutteridge**
Some changes based on our last retro:
* Added images for variables, lists and dictionaries before introducing code
* Removed some difficult exercises, replaced with easier ones

### CODE REVIEW

File: lesson3.html [hunk 1]
```diff
@@ -38,92 +38,101 @@ <h3>Week 3:<br>Lists</h3>
...
         </section>
 
+        <section>
+            <h3>Lists</h3>
+            <p>
+              A list acts as a single variable to store lots of data.
+              <br><br>
+              Each of those values is called an element of the list and can be accessed by its position, or <b>index</b>.
+              <br><br>
+              <img src="static/img/list.png">
```
Comment (AnanaMJ) [inline lesson3.html:48]: Very nice! This will definitely be helpful!

File: lesson6.html [hunk 2]
```diff
@@ -62,53 +76,48 @@ <h3>Dictionaries</h3>
         <section>
           <h3>Dictionaries</h3>
           <p>
-            The indices are called <b>keys</b>.<br>
-            Each key has a <b>value</b> associated with it.<br>
-            So the elements are called <b>key-value pairs</b>.<br><br>
-
+            You can also create a dictionary with data in it:
             <div class="fragment">
```
Comment (AnanaMJ) [inline lesson6.html]: If this is here, the code will show up after you click next:
image
and then
image

And the same for line 100

Comment (agutteridge) [reply lesson6.html]: nice catch Ana, thanks! and added fragments for the questions :)

**Review by AnanaMJ — APPROVED** [review]

**Comment by abhimanyugupta07** [comment]
Is there a way to see these changes live on the slides ?

**Comment by AnanaMJ** [comment]
@abhimanyugupta07 Yep; you can clone the repo and checkout this branch, then open the html locally :)

File: lesson1.html [hunk 3]
```diff
@@ -96,16 +96,29 @@ <h3>Installation</h3>
         <section>
           <h3>Variables</h3>
           <p>
-            Variables are containers for your data<br><br>
+            <b></b>Variables are containers for your data</b><br><br>
             The name of the variable is chosen by the software developer (that’s you!)<br><br>
-            Feel free to write the following in a new file (e.g. monty.py)
+            <div style="margin:auto;width:500px;height:100px;border:1px solid #000;">my name = 'Monty'</div>
```
Comment (abhimanyugupta07) [inline lesson1.html]: having my_name & my_age here would be better so that students don't think you can have `my name` as a variable name.

Comment (abhimanyugupta07) [reply lesson1.html]: I know we have an example negating this after two slides but I feel the introduction has the most impact and it should be the correct syntax :)

Comment (agutteridge) [reply lesson1.html]: good shout, done!

**Review by abhimanyugupta07 — APPROVED** [review]

**Pull request merged**